resource-managers/kubernetes/core/src/test/scala/org/apache/spark/deploy/k8s/features/MountVolumesFeatureStepSuite.scala

```diff
@@ -95,7 +95,7 @@ class MountVolumesFeatureStepSuite extends SparkFunSuite {
       "/tmp",
       "",
       true,
-      KubernetesPVCVolumeConf("pvc-spark-SPARK_EXECUTOR_ID", Some("fast"), Some("512mb"))
+      KubernetesPVCVolumeConf("pvc-spark-SPARK_EXECUTOR_ID", Some("fast"), Some("512m"))
```

Comment:
K8s expects `M` or `Mi`. Since K8s client 3.6.1 enforces this check, I fixed this bug in this PR.

Reply:
Should it be "512M" instead of "512m" ?  
"Pay attention to the case of the suffixes. If you request 400m of ephemeral-storage, this is a request for 0.4 bytes. Someone who types that probably meant to ask for 400 mebibytes (400Mi) or 400 megabytes (400M)." https://kubernetes.io/docs/concepts/configuration/manage-resources-containers/

Reply:
Thank you for review, @bjornjorgensen